packages/pagination/src/PaginationContainer.spec.tsx

```diff
@@ -81,15 +81,15 @@ describe('PaginationContainer', () => {
     );
   };
 
-  const pageProps = props => {
+  const pageProps = (props: IGetPageProps<any>) => {
```

Comment:
Alternatively, could type this function like the snippet below which passes `Item` to `IGetPageProps` generic, but I did not see a huge benefit. Please let me know if this is not the case, or if there is a better way to type the `pageProps`  function.

```tsx
// alternative type annotation
const pageProps = <Item extends {}>(props: IGetPageProps<Item>) => {
  return render(
    <PaginationContainer>
      {({ getPageProps }) => <div {...getPageProps({ 'data-test-id': 'page', ...props })} />}
    </PaginationContainer>
  );
};
```